guides/common/modules/proc_enabling-rhsm-debug-logging.adoc

```diff
@@ -0,0 +1,26 @@
+[id="enabling-rhasm-debug logging_{context}"]
+= Enabling debug logging
+
+You can enable debug logging for the `/var/log/rhsm/rhsm.log` file by using the {ProjectWebUI} or the Hammer CLI tool.
+
+After you resolve the problem, you must disable debug logging and redeploy the virt-who configuration.
```

Comment:
```suggestion
After you resolve the problem, you must disable debug logging and redeploy the virt-who configuration.
To use the CLI instead of the {ProjectWebUI}, see the xref:cli-enabling-rhsm-debug-logging[].

```